pkgs/applications/virtualization/toolbox/default.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, go-md2man
+, podman
+}:
+buildGoModule rec {
+  pname = "toolbox";
+  version = "0.0.99";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "toolbox";
+    rev = version;
+    sha256 = "CKjndWXDLxQ5epco/+TwRFxftbeMhYp/7NOWglF6EL8=";
+  };
+  sourceRoot = "source/src";
```

```suggestion
  modRoot = "src";
```
modRoot should work for this.